src/locale/uz.js

```diff
@@ -1,14 +1,14 @@
-// Uzbek [uz]
+// Uzbek Latin [uz]
```

Comment:
this update is a BREAKING CHANGE to all our `uz` local users.  We have no plan to merge it in the v1.x release.

Discussion is welcomed, maybe we can update it in the next major release.